.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "cpp/memgraph"]
 	path = cpp/memgraph
 	url = https://github.com/memgraph/memgraph.git
-	branch = release/2.8
+	branch = fix-creating-mgp-value-from-nullptr
```

Comment:
This will be rewritten to release/v2.9 once it's up

Reply:
Yes, we can bump this up then.